src/fsharp/TypedTreeOps.fs

```diff
@@ -5695,6 +5774,32 @@ let mkArrayType (g: TcGlobals) ty = TType_app (g.array_tcr_nice, [ty])
 
 let mkByteArrayTy (g: TcGlobals) = mkArrayType g g.byte_ty
 
+//---------------------------------------------------------------------------
+// Witnesses
+//---------------------------------------------------------------------------
+
+let GenWitnessArgTys (g: TcGlobals) (traitInfo: TraitWitnessInfo) =
+    let (TraitWitnessInfo(_tys, _nm, _memFlags, argtys, _rty)) = traitInfo
+    let argtys = if argtys.IsEmpty then [g.unit_ty] else argtys
+    let argtysl = List.map List.singleton argtys
+    argtysl
+    //match tys with 
```

Comment:
This commented code should probably deleted.

Reply:
done thanks